src/pages/signin/SignInPageLayout/index.js

```diff
@@ -18,26 +18,22 @@ const propTypes = {
     ...windowDimensionsPropTypes,
 };
 
-const SignInPageLayout = props => (
-    !props.isSmallScreenWidth
-        ? (
-            <SignInPageLayoutWide
-                welcomeText={props.welcomeText}
-                isMediumScreenWidth={props.isMediumScreenWidth}
-                shouldShowWelcomeText={props.shouldShowWelcomeText}
-            >
-                {props.children}
-            </SignInPageLayoutWide>
-        )
-        : (
-            <SignInPageLayoutNarrow
-                welcomeText={props.welcomeText}
-                shouldShowWelcomeText={props.shouldShowWelcomeText}
-            >
-                {props.children}
-            </SignInPageLayoutNarrow>
-        )
-);
+const SignInPageLayout = (props) => {
+    const content = (
+        <SignInPageContent
+            welcomeText={props.welcomeText}
+            shouldShowWelcomeText={props.shouldShowWelcomeText}
+        >
+            {props.children}
+        </SignInPageContent>
+    );
+
+    if (props.isSmallScreenWidth) {
+        return content;
+    }
+
+    return <SignInPageWideContainer>{content}</SignInPageWideContainer>;
```

Comment:
Is there really an advantage to having the `SignInPageWideContainer` be its own component? I think it'd be better for us to get rid of the `SignInPageWideContainer` altogether and have the View and SVG components it contains live here in this file instead. In other words, why have these as two files when it's not really complicated to have them be one?

Reply:
It looks clean this way but let me know I can merge.

Reply:
I just merged. Thanks for the suggestion.